pkg/kn/commands/service/export_test.go

```diff
@@ -35,14 +35,15 @@ type expectedServiceOption func(*servingv1.Service)
 type expectedRevisionOption func(*servingv1.Revision)
 type expectedServiceListOption func(*servingv1.ServiceList)
 type expectedRevisionListOption func(*servingv1.RevisionList)
+type expectedKNExportOption func(*clientv1alpha1.Export)
 type podSpecOption func(*v1.PodSpec)
 
 type testCase struct {
-	name                 string
-	latestSvc            *servingv1.Service
-	expectedSvcList      *servingv1.ServiceList
-	revisionList         *servingv1.RevisionList
-	expectedRevisionList *servingv1.RevisionList
+	name             string
+	latestSvc        *servingv1.Service
+	expectedSvcList  *servingv1.ServiceList
+	revisionList     *servingv1.RevisionList
```

Comment:
Shouldn’t this be named `expectedRevList` for completeness And consistency?

Reply:
revisionList is for client recording